src/lib/utils/hooks/SynapseAPI/useEntityBundle.ts

```diff
@@ -29,7 +29,7 @@ export default function useGetEntityBundle(
 ) {
   const { accessToken } = useSynapseContext()
   return useQuery<EntityBundle, SynapseClientError>(
-    [accessToken, 'entitybundle', entityId, version, bundleRequest],
+    [accessToken, 'entity', entityId, 'bundle', version, bundleRequest],
```

Comment:
Thinking forward, our query keys should make sense when invalidating the cache in response to a mutation.

For example, when an entity is mutated, we would invalidate the query key `[accessToken, 'entity', entityId], which (I think) would blow away the bundle, plus whatever other future queries are nested in this way